docs/user/convert/parsing_and_validation.rst

```diff
@@ -68,7 +68,7 @@ must be registered through a special
     }
 
 
-Valdiators are provided with a `Dropwizard MetricRegistry <https://metrics.dropwizard.io/>`__, which can be used
+Validators are provided with a `Dropwizard MetricRegistry <https://metrics.dropwizard.io/>`__, which can be used
```

Comment:
Nice catch.  Makes me wonder how hard it'd be to run a spell checker on the docs directory,